examples/cpp/cmake/Findgeode-native.cmake

```diff
@@ -0,0 +1,102 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
+# file Copyright.txt or https://cmake.org/licensing for details.
+
+#.rst:
+# FindGEODE_NATIVE
+# ---------
+#
+# Find the native Expat headers and library.
+#
+# Imported Targets
+# ^^^^^^^^^^^^^^^^
+#
+# This module defines the following :prop_tgt:`IMPORTED` targets:
+#
+# ``GEODE_NATIVE::GEODE_NATIVE``
```

Comment:
This does not match the target created below.

Reply:
fixed